main/bind/APKBUILD

```diff
@@ -174,30 +182,58 @@ dev() {
 	mv "$pkgdir"/usr/bin/isc-config.sh "$subpkgdir"/usr/bin/
 }
 
-_py3() {
-	pkgdesc="A module allowing rndc commands to be sent from Python programs"
-	depends="$_py3deps"
+plugins() {
+	pkgdesc="The ISC DNS server plugins"
+	depends="$depends_plugins"
+
+	mkdir -p "$subpkgdir"/usr/lib
+	mv "$pkgdir"/usr/lib/bind "$subpkgdir"/usr/lib/
+}
+
+py3_isc() {
+	pkgdesc="Python functions to support BIND utilities"
+	depends="$depends_py3_isc"
+	provides="py3-isc=${_isc_ver}-r${_isc_rel} py3-$pkgname=$pkgver-r$pkgrel"
+	replaces="py3-$pkgname"
+
 	mkdir -p "$subpkgdir"/usr/lib
 	mv "$pkgdir"/usr/lib/python3* "$subpkgdir"/usr/lib/
 }
 
-_dnssec_tools() {
-	pkgdesc="Utilities for DNSSEC keys and DNS zone files management"
-	depends="py3-$pkgname=$pkgver-r$pkgrel"
+pytnon_tools() {
```

Comment:
I wonder how the CI managed to return successful build. `pytnon_tools` is misspelled.

Reply:
They are matched up with the same misspelling.